index.html

```diff
@@ -26,50 +26,275 @@
+          <h1 class="intro">
+            I'm Blessing. <br />
+            Glad to see <br />
+            you!
+          </h1>
+          <p class="intro-info">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+          <div>
+            <p class="connect">let's connect</p>
+            <ul class="social-icon">
+              <li>
+                <a href="https://twitter.com/toonmystory"
+                  ><img src="./images/twitter.svg" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/blessing-asuquo-4509981a2/"
+                  ><img src="./images/linkedin.png" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://medium.com/@blessingmike334"
+                  ><img src="./images/medium.svg" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://github.com/blessing-michael"
+                  ><img src="./images/github.svg" alt="" class="icon-img"
+                /></a>
+              </li>
+              <li>
+                <a href="https://github.com/blessing-michael"
+                  ><img src="./images/Vector (3).svg" alt="" class="icon-img"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </article>
+      </section>
+      <section class="meee">
+      <section class="section-two">
+        <article class="article-1">
+          <img src="./images/work-1.png" alt="" class="article-img" />
+          <div class="info-div1">
+            <h2 class="title">tonic</h2>
+            <div class="info">
+              <h4 class="canopy">canopy</h4>
+              <p class="profession">
+                <span class="prof-span">
+                  <img src="./images/Counter.png" alt="" /></span
+                >back end dev
+              </p>
+              <p class="profession">
+                <span class="prof-span">
+                  <img src="./images/Counter.png" alt="" /></span
+                >2015
+              </p>
+            </div>
+            <div class="info-2">
+              <p class="info-p">
+                A daily selection of privately <br />
+                personalized reads; no accounts or sign-ups required.
+              </p>
+            </div>
+
+            <div class="ul">
+              <ul class="tech-stack">
+                <li class="tech-name">HTML</li>
+                <li class="tech-name">CSS</li>
+                <li class="tech-name">Javascript</li>
+              </ul>
+            </div>
+
+            <button class="project-btn">see project</button>
```

Comment:
- [ ] Please 🙏🏽 use capital letters for a better UI.

| Figma design | Your design |
|---|---|
| ![image](https://github.com/blessing-michael/portfolio/assets/58642949/875dea5c-c170-43c6-84cb-fbed6eff1823) | ![image](https://github.com/blessing-michael/portfolio/assets/58642949/9d7effb6-d9ec-40cb-b2a0-02e4d6086b93) |